show_client/chassis_module_cli.go

```diff
@@ -224,6 +231,141 @@ func getChassisModuleStatusByModule(moduleName string) ([]byte, error) {
 	return jsonBytes, nil
 }
 
+// create database queries for chassis module midplane data
+func createChassisModuleMidplaneQueries(moduleName string) dbQueries {
+	queries := dbQueries{
+		State: [][]string{{StateDB, chassisMidplaneTable}},
+	}
+
+	if moduleName != "" {
+		queries.State[0] = append(queries.State[0], moduleName)
+	}
+
+	return queries
+}
+
+// Get and parse midplane data from database
+func getChassisModuleMidplaneData(queries dbQueries) (map[string]interface{}, error) {
+	// Get state data
+	stateDataBytes, err := GetDataFromQueries(queries.State)
+	if err != nil {
+		log.Errorf("Unable to get midplane state data from queries %v, got err: %v", queries.State, err)
+		return nil, fmt.Errorf("failed to get midplane state data: %w", err)
+	}
+	log.V(2).Infof("Midplane state data bytes: %s", string(stateDataBytes))
+
+	// Parse state data
+	var stateData map[string]interface{}
+	if err := json.Unmarshal(stateDataBytes, &stateData); err != nil {
+		log.Errorf("Failed to unmarshal midplane state data: %v", err)
+		return nil, fmt.Errorf("failed to unmarshal midplane state data: %w", err)
+	}
+
+	return stateData, nil
+}
+
+// create ChassisModuleMidplaneStatus from flat data structure
+func createModuleMidplaneStatusFromFlatData(moduleName string, stateData map[string]interface{}) ChassisModuleMidplaneStatus {
+	module := ChassisModuleMidplaneStatus{
+		Name: moduleName,
+	}
+
+	// Process state data
+	for key, value := range stateData {
+		if strValue, ok := value.(string); ok {
+			switch key {
+			case "ip_address":
+				module.IPAddress = strValue
+			case "access":
+				module.Reachability = strValue
+			}
+		}
+	}
+
+	return module
+}
+
+func getChassisModuleMidplaneStatus(options sdc.OptionMap) ([]byte, error) {
+	log.V(2).Infof("getChassisModuleMidplaneStatus: called with options: %v", getOptionsKeys(options))
+
+	// Check if a specific module is requested
+	if moduleStr, ok := options["dpu"].String(); ok {
+		log.V(2).Infof("getChassisModuleMidplaneStatus: filtering for module: %s", moduleStr)
+		return getChassisModuleMidplaneStatusByModule(moduleStr)
+	}
+
+	// Get data for all modules
+	log.V(2).Infof("getChassisModuleMidplaneStatus: getting all modules")
```

Comment:
lower this log level

Reply:
I increased the log level for other entry/exit logs with no argument given.